packages/mds-db/geographies.ts

```diff
@@ -65,6 +61,46 @@ export async function readGeographies(params?: {
     throw err
   }
 }
+
+export async function readGeographiesSummary(params?: {
```

Comment:
Minor nit: Should this be `readGeographySummaries`? `readGeographiesSummary` sounds like something that would return me a single summary object.